src/glfw3.jl

```diff
@@ -631,3 +632,33 @@ function standard_window_hints()
 		(AUX_BUFFERS,  0)
 	]
 end
+
+struct GLFWImage
+	width::Cint
+	height::Cint
+	pixels::Ptr{UInt8}
+	gc_ref::Any
```

Comment:
What is this for?

Reply:
I wanted to be on the safe side for not gc'ing the input image... But after reading up a bit on those threads, I think it's save to do this in the cconvert without keeping an extra reference...